snippets/go/example/sample.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"netbackup"
+	"os"
+)
+
+func main() {
+	fmt.Printf("Welcome to sample Go program to call NetBackup APIs\n")
+	scanner := bufio.NewScanner(os.Stdin)
+	fmt.Printf("Enter the server to connect: ")
+	scanner.Scan()
+	server := scanner.Text()
+	fmt.Printf("Checking connectivity with NetBackup Server[%s]\n", server)
+	fmt.Printf("-----------------------\n")
+	fmt.Println(netbackup.Ping(server))
+	fmt.Printf("-----------------------\n")
+	fmt.Printf("Enter username for server:")
+	scanner.Scan()
+	username := scanner.Text()
+	fmt.Printf("Enter password for server:")
+	scanner.Scan()
+	password := scanner.Text()
+	fmt.Printf("Calling NetBackup Login\n")
+	fmt.Printf("-----------------------\n")
+	jwt := netbackup.Login(username, password, server)
```

Comment:
Would it be possible to create two independent go snippets: `get_nb_images.go` and `get_nb_jobs.go` which demonstrate those 8.1.1 APIs (just like other language snippets) instead of `sample.go`?

Common constants and functionality such as login can remain in `netbackup.go` like you have it currently.

Reply:
Sure I will work on these two samples